packages/sdk/src/conversation/middleware.ts

```diff
@@ -89,7 +89,7 @@ export class NotificationMiddleware implements Middleware {
 }
 
 export class CommandResponseMiddleware implements Middleware {
-  private commandHandlers: TeamsFxBotCommandHandler[] = [];
+  public commandHandlers: TeamsFxBotCommandHandler[] = [];
```

Comment:
If public, does it need to be `readonly`?

Reply:
`readonly` will be better to avoid some unexpected assignment (e.g. `undefined`) that break the register functionality after initialization.